doc/source/GettingStarted/TutorialAssemblingTheGafferBot/index.md

```diff
@@ -432,7 +432,7 @@ Begin by creating another shader:
 
 1. Create another pair of Disney Surface shader and Closure2Surface nodes.
 
-2. Connect the outColor plug of the Disney Shader node to the input plug of the Closure2Surface node.
+2. Connect the OutColor plug of the Disney Shader node to the input plug of the Closure2Surface node.
```

Comment:
Its actually a crazy name now
![image](https://user-images.githubusercontent.com/896779/72746642-7fa1a300-3baa-11ea-8424-6cd476d39f90.png)

Reply:
Not in scope, but should we move away from the Disney shader?

Reply:
Any particular reason?

Reply:
With the updates to AS's Standard Surface, I'm wondering if it's more in line with what Arnold users will expect?